Bonsai.Tinkerforge/Bonsai.Tinkerforge.csproj

```diff
@@ -15,6 +15,7 @@
     <TargetFramework>net472</TargetFramework>
     <Features>strict</Features>
     <Version>0.1.0</Version>
+    <DebugType>embedded</DebugType>
```

Comment:
I was checking the [code generation options](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#debugtype) to understand this flag a bit better, and apparently what it does is embed the PDB inside the DLL itself.

For referencing remote builds in other projects we've been using so far the default setting and generating debug packages instead (with the [`.snupkg` format](https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg)). Since the `embedded` compiler option seems to generate around ~20% larger DLLs, I would suggest we remove it for now to keep packages smaller and for consistency with other releases. In theory for local builds the default should already give you access to full debug information at least for the Debug configuration.

It would be really interesting however to understand and review best practices to have fully integrated symbol debugging even of packages which are downloaded from nuget.org, which I never fully reviewed for .snupkg (there was an even older standard before which is now deprecated). This would improve developer experience across the board for anyone trying to learn how to develop custom bonsai packages, so maybe we can make a note to follow up on this exploration separately.